### ISSUE TITLE
clipcat-menu set loglevel? 

### ISSUE THREAD

**Issue by jonaz** [Icelk/clipcat#20]
I see no config option or cli flag to set the level?
Currently when i copy sensitive data its added to the systemd journal which is not desirable

**Comment by Icelk** [comment]
There is no option to set the "secrecy" of copied text. You can however set `clipcatd`s log_level in the config file. Setting it to `warn` should reject all copy messages. If we assume you're using `clipcat-menu` on the commandline, you can pipe it's output to `/dev/null` if you don't want the clipboard contents to be printed: `clipcat-menu <options> 2>/dev/null`.

Adding a `log_level` option to `clipcat-menu`, like the one available in `clipcatctl` could happen with enough interest.

**Comment by jonaz** [comment]
clipcat-menu still writes to journal even if i pipe the output to /dev/null? 

https://github.com/Icelk/clipcat/blob/main/src/bin/clipcat-menu/command.rs#LL123C24-L123C24

the terminal on top is `journalctl -f`
![2023-05-17-131714_1070x572_scrot](https://github.com/Icelk/clipcat/assets/1146389/170d1d61-3cd6-46a3-aa7a-7c1b18812640)

**Comment by Icelk** [comment]
Right, sorry, you also need to pipe stdout to /dev/null: `clipcat-menu 2>/dev/null 1>&2`

**Comment by jonaz** [comment]
@Icelk same thing because clipcat-menu writes to the journal: 
https://github.com/Icelk/clipcat/blob/main/src/bin/clipcat-menu/command.rs#LL123C24-L123C24

This is why log level needs to be available or journal logging removed. 

`clipcat-menu 2>/dev/null 1>&2`

gives this in journalctl with json for extra details:
```
{"_SYSTEMD_USER_SLICE":"-.slice","_SOURCE_REALTIME_TIMESTAMP":"1684327682044977","_CAP_EFFECTIVE":"0","_SYSTEMD_OWNER_UID":"1000","_SYSTEMD_SESSION":"c16","_TRANSPORT":"journal","_SYSTEMD_SLICE":"user-1000.slice","__REALTIME_TIMESTAMP":"1684327682045033","_SYSTEMD_CGROUP":"/user.slice/user-1000.slice/session-c16.scope","_EXE":"/home/jonasfalck/.cargo/bin/clipcat-menu","TARGET":"clipcat_menu::command","CODE_FILE":"src/bin/clipcat-menu/command.rs","PRIORITY":"5","_HOSTNAME":"FNOX2103","_MACHINE_ID":"811ce057ed1045619f476fa639cf5bb4","_SELINUX_CONTEXT":"unconfined\n","MESSAGE":"Inserting clip (index: 0, id: e37ca0bfd3df9097, content: \"clipcat-menu\")","SYSLOG_IDENTIFIER":"clipcat-menu","_UID":"1000","_GID":"1000","CODE_LINE":"208","__MONOTONIC_TIMESTAMP":"1140685967151","_CMDLINE":"clipcat-menu","_PID":"1749620","_BOOT_ID":"554c657ef09146d0b9962bdbb912eab4","__CURSOR":"s=83050b9dbcd14b13830df2adee367607;i=1f97d;b=554c657ef09146d0b9962bdbb912eab4;m=109962e8f2f;t=5fbe3192c7469;x=80c1061ba1ab5c33","_COMM":"clipcat-menu","_SYSTEMD_UNIT":"session-c16.scope","_SYSTEMD_INVOCATION_ID":"2a0a409d8005488bbbcd41a0b752597c"}
```


TLDR; the problem is not clipcat-menu's output to STDERR and STDOUT. But that it always writes to journal and its not configurable.

**Comment by Icelk** [comment]
Right. Thanks for clarifying. I'll work on it tomorrow.

**Comment by Icelk** [comment]
There is now a `--log-level` option. (set to `warn` to remove copy messages). The environment variable `RUST_LOG` is also available.
